src/main/java/net/datafaker/sequence/FakeSequence.java

```diff
@@ -34,7 +34,7 @@ public T singleton() {
         return null;
     }
 
-    public static abstract class Builder<T> {
+    public abstract static class Builder<T> {
```

Comment:
why?

Reply:
To match the ordering of all other modifiers in the project.

Reply:
I didn't get it
here you proposed the opposite ordering
https://github.com/datafaker-net/datafaker/pull/874/files#diff-87ea11ea16a9ecf7545c61f2ea1f7ed0d98278075881a2359529da3db54683b6R46

Reply:
I can't find the original Sun/Oracle document but this document is based on it; http://yohanan.org/steve/projects/java-code-conventions#SECTION00074000000000000000

which says:

6.4 ORDERING OF MODIFIERS
Though Java allows modifiers in any order, a consistent ordering improves readability of code. The following is the proper order of modifiers for declarations.

First any access modifiers (public, protected, or private) followed by (**in order, if present**) abstract, static, transient, volatile, synchronized, final, and native.

Reply:
Ok, thanks for clarification

then there is another question: wouldn't it be better to set up a ruleset with automation of checks and fix all violations rather than constantly having someone coming with (hope same) set of rules and fixing it?

Reply:
Yes, but we could also say something is better than nothing 😉

Reply:
i'm about +0 to this change meaning:
1 . currently i'm not going to block it, so you can merge it in case you think it is helpful
2. however from my point of view it doesn't solve the main problem: it doesn't prevent same violations in future

Reply:
I don't mind it

Reply:
then at some point the whole repo will be full of commits changing nothing but the order of modifiers...

Reply:
Wouldn't that be a bit of a False Dilemma? 

This if the first time this happens, if someone in the future wants to rearrange things alphabetically, we can always say no.

Having said that, I'm also happy to reject this PR, I see zero value in reordering keywords.

Okay: @dbmalkovsky , please revert the reordering of keywords, there's no significant value in it, and I don't want to enforce the sequence of these keywords in the future.

Reply:
Done